src/http/http_endpoint.h

```diff
@@ -273,9 +282,18 @@ namespace http
       }
       catch (const std::exception& e)
       {
-        send_response(
-          fmt::format("Exception:\n{}\n", e.what()),
-          HTTP_STATUS_INTERNAL_SERVER_ERROR);
+        if (is_websocket)
```

Comment:
Why is there a special case for ws and http here while there isn't on line 263 for example?

Reply:
At 263 and most places we can, we delegate error serialisation to the context (rpc_ctx->serialise_error), but if we fail to create the context (:207) or if the context is gone (:285), we can't do that.

This is a particularly happy situation, if you have a better idea, I'm very happy to explore it!